Magento.gitignore

```diff
@@ -85,6 +86,8 @@ lib/Zend/
 lib/Cm/
 lib/Credis/
 lib/Magento/
+lib/Net/
+dev/
```

Comment:
Why aren't we starting all git ignore lines with a leading slash? There's a good post on what happens without leading slashes.

In your commit, any directory named `dev` would be excluded. Even if it was `/in/another/directory/dev/` which is probably unintended.